Plugins/GRPCSwiftPlugin/plugin.swift

```diff
@@ -205,6 +211,12 @@ struct GRPCSwiftPlugin {
 
       // Add the outputPath as an output file
       outputFiles.append(protobufOutputPath)
+
+        if invocation.reflection == true {
+            let reflectionFile = file.replacingOccurrences(of: "grpc.swift", with: "grpc.reflection")
+            let reflectionOutputPath = outputDirectory.appending(reflectionFile)
+            outputFiles.append(reflectionOutputPath)
+        }
```

Comment:
The indentation is off for this block, it looks like our formatting checker script wasn't updated to include "Plugins" directory, it should be after https://github.com/grpc/grpc-swift/pull/1845

Reply:
Updated